service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterMapper.java

```diff
@@ -18,6 +28,13 @@
     private static final Mustache ENCOUNTER_STATEMENT_TO_EHR_COMPOSITION_TEMPLATE =
         TemplateUtils.loadTemplate("ehr_encounter_to_ehr_composition_template.mustache");
     private static final String COMPLETE_CODE = "COMPLETE";
+    private static final String SNOMED_SYSTEM = "http://snomed.info/sct";
+    private static final String EHR_COMPOSITION_SNOMED_CODE = "<code code=\"%s\" displayName=\"%s\" codeSystem"
```

Comment:
It would be a bit cleaner to create a single mustache template for this, or incorporate into the template for ehrComposition.